assets/src/conversion/index.js

```diff
@@ -133,23 +190,46 @@ class Conversion extends Component {
 							</b>
 						</li>
 					</ul>
+
 					<br />
 
 					<input
 						type="submit"
 						className="large"
 						id="convert_button"
-						value={ __( 'Start Conversion Now' ) }
+						value={ __( 'Start Conversion' ) }
 						onClick={ event => this.handleOnClickInitializeConversion( event ) }
 					/>
-					<p>
-						<i>
-							*{' '}
-							{ __(
-								'note carefully -- once started, the conversion should not be interrupted! Your browser page needs to remain active until conversion is complete. You may, however, manually open multiple browser tabs once the conversion starts, and each of them will fetch and convert another batch of posts in parallel. Running multiple tabs speeds up the conversion. Recommended max number of tabs is 4.'
-							) }
-						</i>
-					</p>
+
+					<br />
+					<br />
+
+					{ !! someConversionsFailed && (
+						<div>
+							<h3>{ __( 'Retry Converting Failed Posts' ) }</h3>
+							<p>
+								{ __(
+									"Looks like some posts weren't converted properly. This could have happened due to unexpected/unsupported Post content (valid HTML source is expected as Post content), or due to technical reasons (e.g. your server experiencing difficulties). You may retry converting these posts now."
+								) }
+							</p>
+							<ul>
+								<li>
+									{ __( 'number of posts which failed getting converted:' ) }{' '}
+									<b>
+										<u>{ countFailedConverting }</u>
+									</b>
+								</li>
+							</ul>
+
+							<input
+								type="submit"
+								className="large"
+								id="retry_failed_button"
```

Comment:
The ids on these inputs don't seem to be used anywhere - shouldn't they be removed?

Reply:
Hmm, for some reason I thought all input elements needed to have explicitly set unique ids... But I see now that's not true (at least not anymore). Updated!